cardano-api/gen/Gen/Cardano/Api.hs

```diff
@@ -92,7 +92,7 @@ genCostModels = do
   CostModel cModel <- genCostModel
   lang <- genLanguage
   case Alonzo.mkCostModel lang cModel of
-    Left err -> panic . Text.pack $ "genCostModels: " <> err
+    Left err -> panic . Text.pack $ "genCostModels: " <> show err
```

Comment:
FYI these errors have a `Pretty` instance if you want to use that.

Reply:
Yep that sounds good